src/PlaywrightSharp.Abstractions/IElementHandle.cs

```diff
@@ -213,5 +213,11 @@ public interface IElementHandle : IJSHandle
         /// </remarks>
         /// <returns>A <see cref="Task"/> that completes when the script finishes or the promise is resolved, yielding the result of the script.</returns>
         Task<T> QuerySelectorAllEvaluateAsync<T>(string selector, string script, params object[] args);
+
+        /// <summary>
+        /// Calls focus on the element.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> that completes when the message is confirmed by the browser.</returns>
+        Task FocusAsync();

```

Comment:
shouldn't this have an option parameter?

Reply:
No, because the `IPage` or the `IFrame` need to wait for the selector. The `IElementHandle` doesn't need to wait for itself.